tests/app/views/contexts/summary/test_question.py

```diff
@@ -2,10 +2,88 @@
 
 from app.data_models import Answer, ListStore
 from app.data_models.answer_store import AnswerStore
+from app.questionnaire import QuestionnaireSchema
 from app.questionnaire.rules.rule_evaluator import RuleEvaluator
 from app.views.contexts.summary.question import Question
 from tests.app.app_context_test_case import AppContextTestCase
 
+ADDRESS_QUESTIONNAIRE_SCHEMA = QuestionnaireSchema(
```

Comment:
Would like this match the pattern in the rest the file (i.e def dynamic_answer_options_schema())